src/components/Calculator/DigitButtons/index.tsx

```diff
@@ -0,0 +1,21 @@
+import styles from "./index.module.css";
+
+interface DigitButtonsPropsType {
+  onClickDigit(value: number): void;
+}
+
+export default function DigitButtons({ onClickDigit }: DigitButtonsPropsType) {
+  return (
+    <div className={`${styles.digits} flex`}>
+      {Array.from({ length: 10 }, (_, idx) => 9 - idx).map((digit) => (
```

Comment:
```suggestion
      {DIGITS.map((digit) => (
```
이 부분은 따로 추출하는 것이 코드를 읽는 데 더 수월할 것 같습니다.